server/webdriver_tests/tests/timeline_test.py

```diff
@@ -80,12 +80,10 @@ def test_charts_from_url_directly_and_uncheck_statvar(self):
     self.driver.get(self.url_ + TIMELINE_URL + URL_HASH_1)
 
     # Wait until the group of charts has loaded.
-    element_present = EC.presence_of_element_located((By.ID, 'chart-region'))
-    WebDriverWait(self.driver, self.TIMEOUT_SEC).until(element_present)
+    shared.wait_for_charts(self.driver)
```

Comment:
doesn't look like wait_for_charts does the waiting? I think this needs to be wrapped in WebdriverWait

Reply:
Fixed